aspnetcore/security/authentication/azure-ad-b2c.md

```diff
@@ -0,0 +1,119 @@
+| **Name** | *\<application name\>* | Enter a **Name** for the application that describes your application to consumers. | 
+| **Include web app / web API** | Yes | |
+| **Allow implicit flow** | Yes | |
+| **Reply URL** | `https://localhost:44300` | Reply URLs are endpoints where Azure AD B2C returns any tokens that your application requests. Visual Studio will provide the Reply URL to use. For now, enter `https://localhost:44300` to complete the form.
+| **App ID URI** | Leave blank | | Not required for this tutorial.
+| **Include native client** | No | |
+
+> [!WARNING]
+> If setting up a non-localhost Reply URL, be aware of the [constraints on what can be added to the Reply URL list](https://docs.microsoft.com/azure/active-directory-b2c/active-directory-b2c-app-registration#choosing-a-web-app-or-api-reply-url). 
+
+After the application registration is created, the list of applications in the tenant is displayed. Select the application that was just registered.  Select the **Copy** icon to the right of the **Application ID** field to copy the Application ID to the clipboard.
+
+Nothing more can be configured in the AAD B2C tenant at this time, but leave the browser window open. There will be more configuration after the ASP.NET Core application is created.
+
+## Create an ASP.NET Core application in Visual Studio 2017
+
+The Visual Studio Web Application template can be configured to use the AAD B2C tenant for authentication.
+
+In Visual Studio:
+
+1. Create a new ASP.NET Core Web Application. 
+2. Select Web Application from the list of templates.
+3. Select the **Change Authentication** button.
+    
+    ![Change Authentication Button](./azure-ad-b2c/_static/changeauth.png)
+
+4. In the **Change Authentication** dialog, select **Individual User Accounts**, and then select **Connect to an existing user store in the cloud** in the dropdown. 
+    
+    ![Change Authentication Dialog](./azure-ad-b2c/_static/changeauthdialog.png)
+
+5. Complete the form with the following values:
+    
+    | Setting      | Value  | 
+    | ------------ | ------- | 
+    | **Domain Name** | *\<the domain name of your B2C tenant\>* | 
+    | **Application ID** | *\<paste the Application ID from the clipboard\>* |
+    | **Callback Path** | *\<use the default value\>* |
+    | **Sign-up or sign-in policy** | `B2C_1_SiUpIn` |
+    | **Reset password policy** | `B2C_1_SSPR` |
+    | **Edit profile policy** | *\<leave blank\>* | 
+
+    Select the **Copy** link next to **Reply URI** to copy the Reply URI to the clipboard.  Select **OK** to close the **Change Authentication** dialog, and then select **OK** to create the web app.
+
+## Finish the B2C application registration
+
+In the browser window, with the B2C application properties still open, change the temporary **Reply URL** specified earlier to the value copied from Visual Studio, and then select **Save** at the top of the window.
```

Comment:
* B2C --> AAD B2C
* Break this into a couple sentences to improve the machine translation